src/ol_infrastructure/applications/edxapp/Pulumi.applications.edxapp.mitx-staging.Production.yaml

```diff
@@ -4,6 +4,21 @@ encryptedkey: AQICAHgQalNS7T35ZlcFdhF0QuKeiJAbXMUbm01pjGwHEsjRCgEgvt/Jaq3CayTQkS
 config:
   aws:region: us-east-1
   consul:address: https://consul-mitx-staging-production.odl.mit.edu
+  edxapp:waffle_flags:
+  - ["course_experience.relative_dates", "--create", "--superusers", "--everyone"]
+  - ["discussions.pages_and_resources_mfe", "--create", "--superusers", "--everyone"]
+  - ["grades.bulk_management", "--create", "--superusers", "--everyone"]
+  - ["grades.enforce_freeze_grade_after_course_end", "--create", "--superusers", "--everyone"]
+  - ["grades.rejected_exam_overrides_grade", "--create", "--superusers", "--everyone"]
+  - ["grades.writable_gradebook", "--create", "--superusers", "--everyone"]
+  - ["learner_home_mfe.enabled", "--create", "--superusers", "--everyone"]
+  - ["learner_recommendations.enable_dashboard_recommendations", "--create", "--superusers",
+    "--everyone"]
+  - ["new_core_editors.use_new_problem_editor", "--create", "--superusers"]
```

Comment:
It's a bit confusing to turn this flag on only for superusers. Is there a reason for that? Can we omit them?